### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

**Comment by Dmytry95** [comment]
image

image

image

image

image

image

File: taxi/urls.py [hunk 1]
```diff
@@ -1,9 +1,23 @@
 from django.urls import path
 
-from .views import index
+from .views import index, ManufacturerListView, CarListView, \
+    CarDetailView, DriverListView, DriverDetailView
```
Comment (vsmutok) [inline taxi/urls.py:4]: better to use `()` then `\`

File: taxi/views.py [hunk 2]
```diff
@@ -17,3 +18,35 @@ def index(request):
...
+    model = Manufacturer
+    context_object_name = "manufacturer_list"
+    queryset = Manufacturer.objects.all().order_by("name")
+    template_name = "taxi/manufacturer_list.html"
+    paginate_by = 5
+
+
+class CarListView(generic.ListView):
+    model = Car
+    queryset = Car.objects.all().select_related("manufacturer")
```
Comment (vsmutok) [inline taxi/views.py:33]: remove redundant `all()`

File: templates/taxi/manufacturer_list.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  {% if manufacturer_list %}
+    <ul>
+      {% for manufacturer in manufacturer_list %}
+        <li>Name: {{ manufacturer.name }}, country: {{ manufacturer.country }}</li>
+      {% endfor %}
+    </ul>
+  {% else %}
+    <p>There are no manufacturers presented</p>
+  {% endif %}
+{% endblock %}
```
Comment (vsmutok) [inline templates/taxi/manufacturer_list.html:14]: add new empty line

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: taxi/urls.py [hunk 4]
```diff
@@ -1,9 +1,24 @@
 from django.urls import path
 
-from .views import index
+from .views import (index, ManufacturerListView, CarListView,
+                    CarDetailView, DriverListView, DriverDetailView
+                    )
```
Comment (vsmutok) [inline taxi/urls.py:5]:
```suggestion
from .views import (
    index, 
    ManufacturerListView, 
    CarListView,
    CarDetailView, 
    DriverListView, 
    DriverDetailView
)
```

**Review by vsmutok — APPROVED** [review]
Great 👍

**Pull request closed without merging**